source/Ship.cpp

```diff
@@ -3331,13 +3336,15 @@ double Ship::IdleHeat() const
 	double coolingEfficiency = CoolingEfficiency();
 	double cooling = coolingEfficiency * attributes.Get("cooling");
 	double activeCooling = coolingEfficiency * attributes.Get("active cooling");
+	double radiativeCooling = coolingEfficiency * RadiativeCooling(attributes.Get("maximum temperature"));
 
 	// Idle heat is the heat level where:
-	// heat = heat * diss + heatGen - cool - activeCool * heat / (100 * mass)
-	// heat = heat * (diss - activeCool / (100 * mass)) + (heatGen - cool)
-	// heat * (1 - diss + activeCool / (100 * mass)) = (heatGen - cool)
-	double production = max(0., attributes.Get("heat generation") - cooling);
+	// heat = heat * diss + heatGen - cool - activeCool * heat / (100 * mass) - radiativeCool
+	// heat = heat * (diss - activeCool / (100 * mass)) + (heatGen - cool - radiativeCool)
+	// heat * (1 - diss + activeCool / (100 * mass)) = (heatGen - cool - radiativeCool)
```

Comment:
Changing the comment in this way is unnecessary, if radiative cooling acts like cooling at the point of overheating. It's cooling 2.0, you can group it under cooling.

What you'd need to actually change is "(100 * mass)" for "(max temp * mass)" or some such